test/helpers/kubectl.go

```diff
@@ -71,6 +71,26 @@ func CreateKubectl(vmName string, log *logrus.Entry) *Kubectl {
 	}
 }
 
+// ExecKafkaPodCmd executes shell command with arguments arg in the specified pod residing in the specified
+// namespace. It returns the stdout of the command that was executed.
+// The kafka producer and consumer scripts do not return error if command
+// leads to TopicAuthorizationException or any other error. Hence the
+// function needs to also take into account the stderr messages returned.
+func (kub *Kubectl) ExecKafkaPodCmd(namespace string, pod string, arg string) (string, error) {
```

Comment:
I would move this function to the test itself. It's a specific function for one test case, and it doesn't make sense to me to have on the helpers. IMHO